src/control/cmd/daos_agent/procmon.go

```diff
@@ -213,22 +221,19 @@ func (p *procMon) handleNotifyPoolConnect(ctx context.Context, request *procMonR
 		go info.monitorProcess(child)
 	}
 
+	p.log.Debugf("%s, connect %s/%s", info, dbgId(request.poolUUID), dbgId(request.poolHandleUUID))
 	info.handles.add(request.poolUUID, request.poolHandleUUID)
 }
 
 func (p *procMon) handleNotifyPoolDisconnect(request *procMonRequest) {
-	p.log.Debugf("Received request to disconnect pool:%s with handle %s, for pid:%d", request.poolUUID, request.poolHandleUUID, request.pid)
```

Comment:
At first glance seems to be a relevant debugging information. Could we keep it at trace level ?

Reply:
We are still logging it, just in a more compact and greppable format (from https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12017/4/artifact/Functional%20Hardware%20Medium/ior/small.py/daos_logs.wolf-253/test_ior_small_daos_agent.log):

```
DEBUG 18:31:55.041370 procmon.go:224: pid:121756 (python3.6), connect 6a926447/18bc792d
DEBUG 18:31:55.042952 procmon.go:236: pid:121756 (python3.6), disconnect 6a926447/18bc792d
```